docs/Team.adoc

```diff
@@ -168,18 +168,37 @@ Area Lead for: UI [2016 May - 2017 May]
 
 '''''
 
-== Contributors
+=== Contributors
 
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aokkhoy[Akshay Narayan]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Abrandonyeoxg[Brandon Yeo]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aweikangchia[Chia Wei Kang]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Acheec[Clarence Chee]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3A500poundbear[Derek]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3AKnewYouWereTrouble[Desmond Ang]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3ANeurrone[Dickson Tan]
 . https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aedmundmok[Edmund Mok]
+. https://github.com/issues?utf8=✓&q=org%3Ase-edu+involves%3Aeugenepeh[Eugene Peh]
```

Comment:
uh... no need to list me below if im still on top? :P

Reply:
Ah yes, I spotted the duplicate earlier but forgot to remove it. SO SORRY ABOUT THAT! 😭